pkg/api/config/config.go

```diff
@@ -179,6 +178,11 @@ func (c *Config) LoadAccessControlConfig(viperInstance *viper.Viper) error {
 
 		var policyGroup PolicyGroup
 
+		if policy == "anonymousaccess" {
```

Comment:
manually test that this works by running a zot with a anonymousaccess true, it should be case insensitive, but let's make sure.